client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -91,43 +91,48 @@ const ProductFormSimpleCard = ( { product, editProduct, translate } ) => {
 		</Card>
 	);
 
-	const renderStock = () => (
-		<Card className="products__product-form-stock">
-			<div className="products__product-manage-stock-wrapper">
-				<FormLabel>{ translate( 'Manage stock' ) }</FormLabel>
-				<div className="products__product-manage-stock">
-					<div className="products__product-manage-stock-checkbox">
-						<FormCheckbox
-							checked={ Boolean( product.manage_stock ) }
-							name="manage_stock"
-							onChange={ toggleStock } />
-					</div>
+	const stockClasses = classNames( {
+		'products__product-form-stock': true,
```

Comment:
It needs moved out of the object and into the first parameter position, but done!